### PR TITLE
Add Functionality for "Exception" Information to Eateries

### PR DESCRIPTION
**Pull request by raahimenon**
## Overview
Add "exceptions" query to the eatery database, starting with Mac's Cafe and Bear Necessities. 
Ex.
1. Mac's: Cash Only After 3PM
2. Bear Necessities: Mobile Order Only

## Changes Made
Exceptions are stored as a string array a JSON file in the static_sources folder, indexed by the slug of the eatery. The "exceptions" field was added to the campus eatery table. Since eatery uses SQLite, arrays cannot be directly stored; I opted to store the exceptions as a string delimited with double semicolons ";;" as I reasoned no exception would ever contain such a string (exception strings are intended to be displayed to the user on the app). To query for exceptions, simply add the `exceptions` field under the `eatery` or `campusEatery` query. 

## Test Coverage
Manual testing. I started with a dummy case on 104West!, which worked and left everything else as the empty string. When I updated the JSON include actual exception information for Mac's and Bear Necessities, the 104West! dummy exception properly went away, and the new ones appeared.

## Next Steps (delete if not applicable)
Data entry for more exceptions. Mac's and Bear Necessities are the ones I immediately knew about, but there are more to be added - this should amount to simple edits to the JSON file.

### CODE REVIEW

**Comment by raahimenon** [comment]
I closed it because I thought I saw an issue (VSCode had automatically removed a comma in a file because of format on save) but when I looked back at it I realized the comma was an extra comma in a function call and wouldn't affect anything.

File: src/eatery_db/campus_eatery.py [hunk 1]
```diff
@@ -244,7 +256,7 @@ def parse_static_op_hours(data_json, eatery_model):
                         )
                     if not new_events:
                         new_operating_hours.append(
-                            (CampusEateryHour(eatery_id=eatery_model.id, date=new_date.isoformat(),), dining_items)
+                            (CampusEateryHour(eatery_id=eatery_model.id, date=new_date.isoformat()), dining_items)
```
Comment (alanna-zhou) [inline src/eatery_db/campus_eatery.py:259]: hm are you sure this doesn't break anything

**Review by alanna-zhou — APPROVED** [review]
looks good :) make sure you deploy this onto the server sometime at night to see if it works before we let it stay on the server

File: requirements.txt [hunk 2]
```diff
@@ -30,5 +30,5 @@ six==1.11.0
 SQLAlchemy==1.3.10
 typing==3.6.6
 urllib3==1.24.2
-Werkzeug==0.15.3
+Werkzeug==0.15.5
```
Comment (yuna878) [inline requirements.txt:33]: Make sure you rebase from master so that `Updated version of Graphene and Werkzeug` <- this commit isn't a part of this PR

**Review by yuna878 — APPROVED** [review]
Looks good! Make sure you squash ur commits before merge and make the commit messages present tense & capitalized for consistency

File: src/eatery_db/campus_eatery.py (hunk 1 above)
Comment (raahimenon) [reply src/eatery_db/campus_eatery.py:259]: It's just an extra comma in a constructor, right? Can that affect something?

Comment (alanna-zhou) [reply src/eatery_db/campus_eatery.py:259]: just checking to make sure you know for sure it works, a comma sometimes could make or break things :((

**Pull request merged**